crs:
  general: {}
  motion_planning:
    pre_move_home: false
    home_position:
      joint_names: [shoulder_pan_joint, shoulder_lift_joint, elbow_joint, wrist_1_joint, wrist_2_joint, wrist_3_joint]
      joint_position: [-0.9644, -1.3617, 2.0724, -0.7108, -0.9640, 0.7994]
    process_path:
      tool_speed: 0.1
      offset_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0] # [px, py, pz, rx, ry, rz] relative to default tool
      retreat_dist: 0.03
      approach_dist: 0.03
      tool_frame: "sander_center_link"
      target_force: 50
    media_change:
      change_time: 300.0 # seconds
      change_pose: [-0.381, -0.202, 1.5, 0.0, -0.786, 1.57] # [px, py, pz, rx, ry, rz] in world coordinates
      joint_names: [shoulder_pan_joint, shoulder_lift_joint, elbow_joint, wrist_1_joint, wrist_2_joint, wrist_3_joint]
      joint_position: [-0.02467, -1.7719, -1.8680, -0.6212, 0.90265, -2.2430]
    preview:
      time_scaling: 1.0
  scan_acquisition: 
    scan_poses:
      - pose: [0.0, -0.50, 1.7, 0.0, -2.35619, 1.57]
      - pose: [0.0, -0.25, 1.7, 0.0, -2.35619, 1.57]
      - pose: [0.0, -0.10, 1.7, 0.5, -2.35619, 1.57]
      - pose: [0.0, 0.0, 1.7, 0.0, -2.35619, 1.57]
      - pose: [0.0, 0.1, 1.7, -0.8, -2.35619, 1.57]
    tool_frame: "camera_link_optical" 
    skip_on_failure: true
  process_execution:
    time_tolerance: 5.0 # seconds
    joint_tolerance: [0.035, 0.035, 0.035, 0.035, 0.035, 0.035] # radians
    cartesian_tolerance: [0.015, 0.015, 0.02, 0.05, 0.05, 0.05]
    force_tolerance: 5
    force_controlled_trajectories: true
  part_registration:
    simulation_pose: [-0.7, 0.0, 1.1, 0.0, 0.0, 0.0]  # used for simulator only
    seed_pose: [0.0, 0.0, 0.5, 0.0, 0.0, 0.0]
    target_frame_id: "world"
    part_file: "null"      # relative to known data directory
    toolpath_file: "null"  # relative to known data directory
  part_rework: {}
